meteor-publish-composite/meteor-publish-composite.d.ts

```diff
@@ -0,0 +1,65 @@
+// Type definitions for meteor-publish-composite
+// Project: https://github.com/englue/meteor-publish-composite
+// Definitions by: Robert Van Gorkom <https://github.com/vangorra>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../meteor/meteor.d.ts" />
+
+declare interface IPublishCompositeConfigN {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files
> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.